README.md

```diff
@@ -20,22 +20,44 @@ If you would like to have a custom integration, feel free to [contact us](https:
 
 ## Features
 
-- GPU-accelerated
-- Configurable LiDAR pattern and range
+|   |   |
+|---|---|
+| **Configurable LiDAR pattern and range** | **High performance** |
+| ![](docs/gif/flexible_pattern.gif) | ![](docs/gif/high_performance.gif) |
+| **GPU-accelerated pointcloud processing** | **Flexible pipeline creation** |
+| ![](docs/gif/pointcloud_processing.gif) | ![](docs/image/flexible_pipeline.png) |
+
+And more:
 - Asynchronous raytracing
+- Removing non-hit points
+- Converting to custom binary output
+- Downsampling
+- Writing to PCD file
 - Optional Gaussian noise (see [documentation](docs/GaussianNoise.md))
 
 ## Usage
 
 An introduction to the RGL API along with an example can be found [here](docs/Usage.md).
 
-## Building manually
+## Building on Ubuntu
 
 1. Install [CUDA Toolkit](https://developer.nvidia.com/cuda-downloads) 11.2+.
 2. Download [NVidia OptiX](https://developer.nvidia.com/designworks/optix/downloads/legacy) 7.2
     1. You may be asked to create Nvidia account to download
-3. If you are on Linux or you have chosen non-standard location on Windows when installing OptiX, you need to export environment variable `OptiX_INSTALL_DIR`.
-4. Install [PCL](https://pointclouds.org/) 1.12 on Windows (on Linux this will be done automatically by using a prepared script).
+3. Export environment variable:
+   1. `export OptiX_INSTALL_DIR=<your-OptiX-path>`.
+4. Use `setup.bash --cmake --make` script.
+   - It will install dependencies from `apt`.
+   - It will run CMake and then make.
+   - You can pass optional CMake and make parameters, e.g.
+     - `./setup.bash --cmake -DCMAKE_BUILD_TYPE=Debug --make -j 16`
+
+## Building on Windows
+
+1. Install [CUDA Toolkit](https://developer.nvidia.com/cuda-downloads) 11.2+.
+2. Download [NVidia OptiX](https://developer.nvidia.com/designworks/optix/downloads/legacy) 7.2
+   - use the default location or set environment variable `OptiX_INSTALL_DIR`
+3. Install [PCL](https://pointclouds.org/) 1.12 on Windows (on Linux this will be done automatically by using a prepared script).
```

Comment:
This section: `on Windows (on Linux this will be done automatically by using a prepared script)` is not needed while build instructions are separated.